keras_core/ops/operation_utils.py

```diff
@@ -11,7 +11,23 @@ def compute_pooling_output_shape(
     padding="valid",
     data_format="channels_last",
 ):
-    """Compute the output shape of pooling ops."""
+    """
+    Computes the output shape of pooling operations.
+
+    Args:
+        input_shape (Tuple[int]): The shape of the input.
```

Comment:
Please follow the style of other docstrings

Reply:
Followed the style of other docstrings